<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_ScaleOptions</class>
 <widget class="QWidget" name="QG_ScaleOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>37</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbInfo">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>10</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>缩放参数:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_manual">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>11</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>手动</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_fixed">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>11</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>固定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>8</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>11</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>基点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>11</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_xBaseValue">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>9</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>11</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yBaseValue">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>9</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>9</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>比例</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>11</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_xFactorValue">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>9</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>11</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yFactorValue">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>9</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_manual</sender>
   <signal>clicked()</signal>
   <receiver>QG_ScaleOptions</receiver>
   <slot>changeSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_fixed</sender>
   <signal>clicked()</signal>
   <receiver>QG_ScaleOptions</receiver>
   <slot>changeSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>QG_ScaleOptions</receiver>
   <slot>scaleFixed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeSelect()</slot>
  <slot>scaleFixed()</slot>
 </slots>
</ui>
